content/en/opentelemetry/interoperability/environment_variable_support.md

```diff
@@ -0,0 +1,122 @@
+      text: '.NET Framework Tracing Configuration'
+    - link: '/tracing/trace_collection/library_config/go'
+      tag: 'Documentation'
+      text: 'Go Tracing Configuration'
+    - link: '/tracing/trace_collection/library_config/java'
+      tag: 'Documentation'
+      text: 'Java Tracing Configuration'
+    - link: '/tracing/trace_collection/library_config/nodejs'
+      tag: 'Documentation'
+      text: 'Node.js Tracing Configuration'
+    - link: '/tracing/trace_collection/library_config/php'
+      tag: 'Documentation'
+      text: 'PHP Tracing Configuration'
+    - link: '/tracing/trace_collection/library_config/python'
+      tag: 'Documentation'
+      text: 'Python Tracing Configuration'
+    - link: '/tracing/trace_collection/library_config/ruby'
+      tag: 'Documentation'
+      text: 'Ruby Tracing Configuration'
+---
+
+After you set up the tracing library with your code and configure the Agent to collect APM data, optionally configure the tracing library as desired using OpenTelemetry environment variables
+
+Tracer settings can be configured with the following environment variables to their equivalent Datadog environment variable mapping:
+
+`OTEL_SERVICE_NAME`
+: **Mapping**: `DD_SERVICE`<br>
+
+`OTEL_LOG_LEVEL`
+: **Mapping**: `DD_LOG_LEVEL`<br>
+**Notes**: A log level of debug will also map to `DD_TRACE_DEBUG=true`<br>
+This maps to `DD_TRACE_LOG_LEVEL` in `node.js`<br>
+**Not Supported In**: `python`, `dotnet`, `ruby`, & `golang` tracers<br>
+
+`OTEL_PROPAGATORS`
+: **Mapping**: `DD_TRACE_PROPAGATION_STYLE`<br>
+**Notes**: `datadog` along with `xray` for the `java` tracer is also accepted<br>
+
+`OTEL_TRACES_SAMPLER & OTEL_TRACES_SAMPLER_ARG`
+: **Mapping**: `DD_TRACE_SAMPLE_RATE`<br>
+**Notes**: Here are what the values passed in map to for `DD_TRACE_SAMPLE_RATE`:<br>
+  - OTEL_TRACES_SAMPLER=`parentbased_always_on`		  DD_TRACE_SAMPLE_RATE=`1.0`
+  - OTEL_TRACES_SAMPLER=`parentbased_always_off`		DD_TRACE_SAMPLE_RATE=`0.0`
+  - OTEL_TRACES_SAMPLER=`parentbased_traceidratio`	DD_TRACE_SAMPLE_RATE=`${OTEL_TRACES_SAMPLER_ARG}`
+  - OTEL_TRACES_SAMPLER=`always_on`				          DD_TRACE_SAMPLE_RATE=`1.0`
+  - OTEL_TRACES_SAMPLER=`always_off`				        DD_TRACE_SAMPLE_RATE=`0.0`
+  - OTEL_TRACES_SAMPLER=`traceidratio`			        DD_TRACE_SAMPLE_RATE=`${OTEL_TRACES_SAMPLER_ARG}`
+
+`OTEL_TRACES_EXPORTER`
+: **Mapping**: `DD_TRACE_ENABLED=false` <br>
+**Notes**: only a value of `none ` is accepted<br>
+
+`OTEL_METRICS_EXPORTER`
+: **Mapping**: `DD_RUNTIME_METRICS_ENABLED=false` <br>
+**Notes**: only a value of `none` is accepted<br>
+**Not Supported In**: `php` tracer<br>
+
+`OTEL_RESOURCE_ATTRIBUTES`
+: **Mapping**: `DD_TAGS` <br>
+**Notes**: `deployment.environment` maps to the `DD_ENV` environment variable<br>
+`service.name` maps to the `DD_SERVICE` environment variable<br>
+`service.version` maps to the `DD_VERSION` environment variable
+
+`OTEL_SDK_DISABLED`
+: **Mapping**: `!DD_TRACE_OTEL_ENABLED` <br>
```

Comment:
I think here it would be better to show them example of how true is false for the other I would love to use `!` but we need to assume the users may have never code or get that.